index.html

```diff
@@ -45,9 +45,7 @@ <h1 class="headTitle">Clarkson's Exotic Auto</h1>
       <div class="mainBody container-fluid">
 
         <!-- Automotive Display Cards -->
-        <div class="row">
-          <div class="col-md-4" id="cardArea">
-          </div>
+        <div class="row" id="cardArea">
```

Comment:
This `div` should be dynamically generated through your js. What happens when you add a fourth car or an eighth?